digitaloceanmodule.php

```diff
@@ -2,25 +2,11 @@
 
 class Digitaloceanmodule extends Module {
 
-    private static $version = "2.2";
-    private static $authors = array(array('name' => "ModulesBakery.com.", 'url' => "http://www.modulesbakery.com"));
```

Comment:
yes they are listed as original author that was moved to config.json I am working on right now.